### ISSUE TITLE
[ngAria] Don't warn about missing aria-label when aria-labelledby is specified

### ISSUE THREAD

**Issue by lyschoening** [angular/angular.js#13758]
ngAria requires "aria-label" even when a valid "aria-labelledby" attribute is defined.

Many of components in our templates that require an "aria-label" attribute are already labeled by blocks of text next to it. I would therefore like to use "aria-labelledby" instead of having to maintain a copy of the text in "aria-label". 

I believe that many developers would see the logic in specifying "aria-labelledby" vs having to maintain an "aria-label" property when the interface already seems self-explanatory to them.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

**Comment by Narretz** [comment]
Sounds reasonable. Is this something you'd be interested to work on?

**Comment by lyschoening** [comment]
Seems I have been mistaken and this is actually an issue with angular-material. Sorry, I falsely assumed mdAria was using ngAria for this.
